### PR TITLE
Docker TaskRunners

### PR DESCRIPTION
**Pull request by royceschultz**

### CODE REVIEW

**Comment by royceschultz** [comment]
ISSUE: Instructions to run in README.md are no longer valid.

File: taskrunners/cleaner/Dockerfile [hunk 1]
```diff
@@ -4,3 +4,5 @@ COPY requirements.txt requirements.txt
 COPY setup_requirements.sh setup_requirements.sh
```
Comment (AndrewYeeYee) [inline taskrunners/cleaner/Dockerfile:4]: I think setup_requirements.sh needs to be added to this directory

File: taskrunners/downloader/src/Connect.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+db = None
+
+def connect():
+    global db
+    domain = 'domain.com'
+    username = 'USERNAME'
+    password = 'PASSWORD'
+    authSource = 'user_table'
+    return MongoClient(domain, 27017, username=username, password=password, authSource=authSource).finlab_prod
```
Comment (AndrewYeeYee) [inline taskrunners/downloader/src/Connect.py:11]: Is there a reason this is finlab_prod? The other was finlab_beta. Connect.py don't connect to the same db.

File: taskrunners/cleaner/src/Connect.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from pymongo import MongoClient
+
+def connect():
+    domain = 'domain.com'
+    username = 'user'
+    password = 'password'
+    authSource = 'auth'
+    return MongoClient(domain, 27017, username=username, password=password, authSource=authSource).finlab_beta
```
Comment (AndrewYeeYee) [inline taskrunners/cleaner/src/Connect.py:8]: Is there a reason this is finlab_beta? The other Connect.py has finlab_prod. Connect.py aren't consistent

**Review by AndrewYeeYee — COMMENTED** [review]
I found a few things inconsistencies with db connections and a missing setup script.

File: taskrunners/downloader/src/Connect.py (hunk 2 above)
Comment (royceschultz) [reply taskrunners/downloader/src/Connect.py:11]: Updated the other to finlab_prod.

File: taskrunners/cleaner/src/Connect.py (hunk 3 above)
Comment (royceschultz) [reply taskrunners/cleaner/src/Connect.py]: Updated to finlab_prod.

File: taskrunners/cleaner/Dockerfile (hunk 1 above)
Comment (royceschultz) [reply taskrunners/cleaner/Dockerfile]: Removed the COPY step. setup_requirements was copied from somewhere else. It's not used for anything.

File: taskrunners/cleaner/Dockerfile [hunk 4]
```diff
@@ -1,8 +1,6 @@
 FROM python:3.8.6
 WORKDIR /usr
 COPY requirements.txt requirements.txt
-COPY setup_requirements.sh setup_requirements.sh
-RUN ./setup_requirements.sh
```
Comment (AndrewYeeYee) [inline taskrunners/cleaner/Dockerfile]: How are the requirements being install now?

Comment (royceschultz) [reply taskrunners/cleaner/Dockerfile]: Oh wait, you're right. This is a different container than I thought. I'll find where that file went.

Comment (royceschultz) [reply taskrunners/cleaner/Dockerfile:5]: Reverted the dockerfile and added the missing setup script.

**Review by AndrewYeeYee — APPROVED** [review]
Looks good to me. There were a lot of changes so it may be worth it to get input from Chace or Tianwei on this.

**Review by chacetrev10 — APPROVED** [review]

**Pull request merged**